symphony-bdk-spring/symphony-bdk-app-spring-boot-starter/src/main/java/com/symphony/bdk/app/spring/exception/GlobalControllerExceptionHandler.java

```diff
@@ -0,0 +1,59 @@
+package com.symphony.bdk.app.spring.exception;
+
+import com.symphony.bdk.core.auth.exception.AuthInitializationException;
+
+import com.symphony.bdk.core.auth.exception.AuthUnauthorizedException;
+
+import org.springframework.context.support.DefaultMessageSourceResolvable;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.context.request.WebRequest;
+import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
+
+import java.time.LocalDateTime;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+@ControllerAdvice
+public class GlobalControllerExceptionHandler extends ResponseEntityExceptionHandler {
+
+  public GlobalControllerExceptionHandler() {
+    super();
+  }
+
+  @ExceptionHandler({
+      AuthInitializationException.class,
+      AuthUnauthorizedException.class
```

Comment:
Those exceptions should be handled and wrapped in a service layer. Then this layer would throw an `AppAuthException extends RuntimeException` specific to this `app-starter`